src/main/java/org/networkcalculus/dnc/network/server_graph/ServerGraph.java

```diff
@@ -741,6 +741,14 @@ public Flow getFlow(int id) throws Exception {
 		}
 	}
 
+	/**
+	 * Finds all the flows that cross at least one server on the path p.
+	 *
+	 * @param p
+	 *            The path on which the returned flows intersect.
+	 * @return Set of flows that cross at least one server on the path.
+	 * @throws Exception
```

Comment:
Lacks description.
In contrast to the clean-up of outdated and possibly wrong comments in other methods, newly added javadocs should be complete.

Reply:
Any suggestions?

Reply:
I traced potential exceptions in the methods called in the body. I think we get exceptions based if those methods are called with an argument set to null.
Overall, this leads to too much effort for this PR, should be done in a separate work item.